scripts/CEE/list-alerts/metadata.yaml

```diff
@@ -0,0 +1,34 @@
+file: script.sh
+name: list-alerts
+description: Script listing the active/pending alerts in alertmanager. It also provide an option to list active silences. 
+author: bdematte
+allowedGroups: 
+  - SREP
```

Comment:
If this has all the groups, why do we need to duplicate it?

Reply:
This is probably something for the repos owner and not something specific to this PR but more on how managed-scripts are working. 
Probably this is a discussion to have in #sd-ism-backplane and card the implementation if we want to change it (even though, I am not sure all scripts will have necessarily the same groups in a given folder).